src/main/java/org/benetech/servicenet/domain/Phone.java

```diff
@@ -60,138 +65,55 @@
     @JsonIgnoreProperties("phones")
     private ServiceAtLocation serviceAtLocation;
 
-    // jhipster-needle-entity-add-field - JHipster will add fields here, do not remove
```

Comment:
Are you sure jhipster won't fail it the needle will be absent? Will it just omit generating getters and setters?